content/cloud/changelog/spin-cloud-v0-6-0.md

```diff
@@ -0,0 +1,34 @@
+title = "Spin Cloud v0.6 Updates"
+template = "changelog_item"
+date = "2024-1-30T12:00:00Z"
+enable_shortcodes = true
+tags = ["spin_cloud_plugin"]
+[extra]
+type= "changelog_post"
+twitter_card_type = "summary_large_image" 
+image = "/static/image/changelog/TODO.jpg" 
+
+---
+
+Today we're announcing the minor release of [`spin cloud` plugin v0.6.0](https://github.com/fermyon/cloud-plugin), which allows the management of key value stores via CLI. 
+
+Highlights include:
+
+* create a key value store with `spin cloud kv create`
+* delete a key value store with `spin cloud kv delete`
+* link a key value store to a Spin app via `spin cloud link kv`
+* unlink a key value store from a Spin app via `spin cloud unlink kv`
+* write a key/value pair to your key value store via `spin cloud kv set`
+
+<img src="/static/image/changelog/TODO" alt="Spin Cloud 0.6.0 changelog">
```

Comment:
```suggestion
<img src="/static/image/changelog/TODO" alt="Spin Cloud 0.7.0 changelog">
```

Reply:
We deleted the changelog from this PR as it was already covered in https://github.com/fermyon/developer/pull/1161